sql/atxs/atxs.go

```diff
@@ -24,11 +24,15 @@ func Get(db sql.Executor, id types.ATXID) (atx *types.ActivationTx, err error) {
 		tickCount := uint64(stmt.ColumnInt64(2))
 		v.SetID(&id)
 		v.Verify(baseTickHeight, tickCount)
+		buf := make([]byte, len(types.NodeID{}))
```

Comment:
i think we should be consistent with other places by using `stmt.ColumnLen(3)` here

Reply:
i haven't noticed but you don't need that at all. 

```go
id := NodeID{}
stmt.ColumnBytes(3, id[:])
```